src/utils.py

```diff
@@ -1,7 +1,7 @@
 import os
 from semver import Version
 from conda.models.match_spec import MatchSpec
-from conda.env.specs import RequirementsSpec
+from conda_env.specs import RequirementsSpec
```

Comment:
This was actually changed here https://github.com/aws/sagemaker-distribution/pull/190 to account for update in conda pkg. So if you update conda version locally, it will work with updated code. We can restrict the environment.yml to make sure higher version is installed or support both imports, but we shouldn't remove the newer syntax.

Reply:
Got it. Removed that commit from this PR